TestProject/ClientApp/src/components/Posts.js

```diff
@@ -1,11 +1,12 @@
 ﻿import React, { useEffect, useState } from 'react';
 import Post from './Post'
 
-function Posts() {
+function Posts({ userLoggedIn, userProfile, userPosts }) {
```

Comment:
A boolean parameter should be a question that can be answered yes or no (true/false).
From what I see here, I suppose that `userPosts` is user posts, and not some parameter.

Reply:
`filterByCurrentUser` for example, probably you can come up with something even better.